checks.d/postgres.py

```diff
@@ -292,6 +314,18 @@ def _get_bgw_metrics(self, key, db):
             metrics = self.bgw_metrics.get(key)
         return metrics
 
+    def _get_replication_metrics(self, key, db):
+        """ Use either REPLICATION_METRICS_9_1 or REPLICATION_METRICS_9_1 + REPLICATION_METRICS_9_2
+        depending on the postgres version.
+        Uses a dictionnary to save the result for each instance
+        """
+        metrics = self.replication_metrics.get(key)
+        if self._is_9_1_or_above(key, db) and metrics is None:
+            self.replication_metrics[key] = dict(self.REPLICATION_METRICS_9_1)
+            if self._is_9_2_or_above(key, db):
+                self.replication_metrics.update(self.REPLICATION_METRICS_9_2)
```

Comment:
shouldn't it be:
```python
 self.replication_metrics[key].update(self.REPLICATION_METRICS_9_2)
```
? 

Also this seems a bit weird, as you're then overriding a constant (self.REPLICATION_METRICS), which in result will be applied to all instances. So your code won't work in the (weird) case where you have multiples instances of postgres running different versions.

I think _get_replication_metrics should just return the replication metrics (as the name of the method seems to indicate)